www/docs/tailwind.config.ts

```diff
@@ -78,8 +78,19 @@ export default {
   ],
   theme: {
     extend: {
+      animation: {
+        blink: 'blink 1.25s step-end infinite',
+      },
+      keyframes: {
+        blink: {
+          '0%': { opacity: '1' },
+          '50%': { opacity: '0' },
+          '100%': { opacity: '1' },
+        },
+      },
       fontFamily: {
-        sans: ['Manrope Variable"', ...defaultTheme.fontFamily.sans],
+        sans: ['inter', 'sans-serif'],
+        mono: ['geistMono', 'serif'],
```

Comment:
I was previously installing fonts with `@fontsource` packages. This way is fine too, but we can remove the old fontsource packages if we aren't using them anymore.